python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/test_asset_defs.py

```diff
@@ -25,6 +25,21 @@ def test_fivetran_asset_keys():
     )
     assert ft_assets[0].keys == {AssetKey(["x", "foo"]), AssetKey(["y", "bar"])}
 
```

Comment:
```suggestion


```

linter nit

Reply:
@benpankow done.

Apologies that I can't run the tests/linting locally. Daniel mentioned that there is some work going into creating a wheel, so hopefully this will mean that I can setup a dev env in the future